app/controllers/admin/feature_flags_controller.rb

```diff
@@ -9,9 +9,8 @@ def index
 
   def update
     authorise FeatureFlag
-    @flags = FeatureFlag.update_all_flags( feature_flag_params )
 
-    if @flags.all?( &:valid? )
+    if FeatureFlag.update_all_flags( feature_flag_params )
```

Comment:
@denny Try stubbing `FeatureFlag.update_all_flags` in the spec to test when this fails, e.g.:

```rb
expect(FeatureFlag).to receive(:update_all_flags).and_return(false)
```